include/crypto.h

```diff
@@ -113,6 +116,7 @@ uint16_t Crypto_Calc_CRC16(uint8_t* data, int size);
 int32_t Crypto_Check_Anti_Replay(SecurityAssociation_t *sa_ptr, uint8_t *arsn, uint8_t *iv);
 int32_t Crypto_Get_ECS_Algo_Keylen(uint8_t algo);
 int32_t Crypto_Get_ACS_Algo_Keylen(uint8_t algo);
+int32_t Crypto_Get_Algo_Keylen(uint8_t algo);
```

Comment:
Never defined? Not needed? delete Crypto_Get_Algo_Keylen. The name alone seems to indicate you should be using the more specific Crypto_Get_ECS_Algo_Keylen, Crypto_Get_ACS_Algo_Keylen functions above.

Reply:
I agree.  I think this is a leftover that I missed.

Reply:
Gone